files/image_config/caclmgrd/caclmgrd

```diff
@@ -503,3 +501,4 @@ def main():
 
 if __name__ == "__main__":
     main()
+
```

Comment:
Blank line at end of file is should be removed. As long as the last line ends with a newline, you're good. That's one thing that using pep8online.com doesn't translate well, because you don't submit the actual file; you copy and paste the content.

Reply:
@jleveque Done. Thanks for clarifying. I also thought that does not look correct.